src/rp2_common/hardware_pio/include/hardware/pio.h

```diff
@@ -755,12 +757,46 @@ static inline void pio_set_irq1_source_mask_enabled(PIO pio, uint32_t source_mas
     }
 }
 
+/*! \brief  Enable/Disable a single source PIO's specified (0/1) IRQ index
```

Comment:
Changing "a single source PIO's specified" to "a single source on a PIO's specified" would provide more consistency with the doxygen for `pio_set_irqn_source_mask_enabled` (and is IMHO more readable).

Reply:
fxd